ClosedXML/Extensions.cs

```diff
@@ -132,6 +132,22 @@ public static String ToProper(this String value)
 
             return value.Substring(0, 1).ToUpper() + value.Substring(1);
         }
+
+        public static String GetPasswordHash(String password)
```

Comment:
Use an extension method, not a static method. Actually, undo this, and merge https://github.com/nomomax/ClosedXML/pull/4